Backend/app/therapy_sessions/serializers.py

```
@@ -0,0 +1,738 @@
+            return {
+                'patient_id': profile.patient_id,
+                'primary_concern': profile.primary_concern,
+                'therapy_start_date': profile.therapy_start_date,
+                'session_frequency': profile.session_frequency,
+                'preferred_session_days': profile.get_preferred_days_list(),
+                'emergency_contact_name': profile.emergency_contact_name,
+                'emergency_contact_phone': profile.emergency_contact_phone,
+                'preferred_language': profile.preferred_language,
+                'connected_at': profile.connected_at,
+            }
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)